README.md

```diff
@@ -23,10 +20,13 @@ If you have any urgent issues regarding this repository please create a support
 | [`platform-sdk`](/packages/platform-sdk)   | [![platform-sdk Version][platform-sdk-icon]][platform-sdk-version]    |
 | [`importapi-sdk`](/packages/importapi-sdk) | [![importapi-sdk Version][importapi-sdk-icon]][importapi-sdk-version] |
 | [`ml-sdk`](/packages/ml-sdk)               | [![ml-sdk Version][ml-sdk-icon]][ml-sdk-version]                      |
+| [`history-sdk`](/packages/history-sdk)     | [![history-sdk Version][history-sdk-icon]][history-sdk-version]       |
```

Comment:
Was it forgotten or was it omitted on purpose?

Reply:
Note sure, but I think it's fine to have it as part of the packages listed in the README.md